docs/reference/configuration/data_types.rst

```diff
@@ -72,167 +61,352 @@ This section only applies if a fixed seed is used:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A stochastic variable is log normally distributed if the logarithm of the
-variable is normally distributed. In other words, if X is normally distributed,
-then Y = exp(X) is log normally distributed.
+The ``LOGNORMAL`` keyword is used to assign a log normal prior to a variable. A variable is considered log normally distributed if the logarithm of that variable follows a normal distribution. 
+In mathematical terms, if :math:`X` is normally distributed, then :math:`Y = e^X` is log normally distributed.
 
-A log normal prior is suited to model positive quantities with a heavy tail
-(tendency to take large values). To set a log normal prior, use the keyword
-LOGNORMAL. It takes two arguments, the mean and standard deviation of the
-*logarithm* of the variable:
+Log normal priors are especially suitable for modeling positive quantities that exhibit a heavy tail, indicating a tendency for the quantity to occasionally take large values.
 
+Syntax
+~~~~~~
 ::
 
-   VAR2   LOGNORMAL  0  1
+  VAR LOGNORMAL <log_mean> <log_standard_deviation>
 
-TRUNCATED_NORMAL
-,,,,,,,,,,,,,,,,,
+Parameters
+~~~~~~~~~~
+- **<log_mean>**: Represents the mean of the logarithm of the variable.
+- **<log_standard_deviation>**: Denotes the standard deviation of the logarithm of the variable.
+
+Example
+~~~~~~~
+For a variable with its logarithm having a mean of 0 and a standard deviation of 1:
+::
+
+   VAR LOGNORMAL 0 1
 
-This *TRUNCATED_NORMAL* distribution works as follows:
+.. image:: fig/lognormal.png
 
-   1. Draw random variable :math:`X \sim N(\mu,\sigma)`
-   2. Clamp X to the interval [min, max]
+TRUNCATED_NORMAL: Truncated Normal Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This is **not** a proper normal distribution; hence the
-clamping to `[min,max]` should be an exceptional event. To configure
-this distribution for a situation with mean 1, standard deviation 0.25
-and hard limits 0 and 10:
+The ``TRUNCATED_NORMAL`` keyword is utilized to assign a truncated normal distribution to a variable.
+This distribution works as follows:
 
+   1. Draw a random variable :math:`X \sim N(\mu,\sigma)`.
+   2. Clamp :math:`X` to the interval [min, max].
+
+It's important to note that this is **not** a proper normal distribution due to the constraints applied.
+Thus, the act of clamping values to the `[min,max]` interval should be an exceptional event. 
+
+Syntax
+~~~~~~
 ::
 
-   VAR3  TRUNCATED_NORMAL  1  0.25   0  10
+  VAR TRUNCATED_NORMAL <mean> <standard_deviation> <min> <max>
 
+Parameters
+~~~~~~~~~~
+- **<mean>**: Specifies the mean of the normal distribution prior to truncation.
+- **<standard_deviation>**: Indicates the standard deviation of the distribution before truncation.
+- **<min>**: Determines the lower truncation limit.
+- **<max>**: Establishes the upper truncation limit.
+
+Example
+~~~~~~~
+Reasonable usage of this distribution where the number of values being clamped is relatively low.
+::
```

Comment:
Why is this reasonable compared to the example below? Or, why is excessive clamping problematic?

Reply:
Not sure about this.
I've heard people talk about this and that it can lead to too high probabilities of end-points.

Reply:
I think the answer is in "more" clamping results in implicit increased non-linearity of the forward model (from gaussian sample --> desired distribution --> forward model result). So indeed the validity of data assimilation linearization can be put more into question with increased clamping. I do not think it will necessarily be worse than other distributions though, e.g. using "errf" and "derrf" we can induce a great deal of non-linearity. Since we are not doing any warnings for non-linear distributions, and we will have non-linearity due to other parts of the forward model anyway, should we explicitly warn against it here?

Reply:
We should perhaps create a document describing the effect of these transformations, but I agree with you that we should not warn about it here.
I've removed it and the figure showing "bad" clamping.